ui-src/package.json

```diff
@@ -4,35 +4,40 @@
   "private": true,
   "proxy": "http://localhost:4141",
   "dependencies": {
-    "react": "16.4.2",
-    "react-dom": "16.4.2",
-    "react-scripts-ts": "2.17.0",
     "@material-ui/core": "latest",
     "@material-ui/icons": "^2.0.2",
     "autosuggest-highlight": "^3.1.1",
-    "hc-redux-middleware": "latest",
```

Comment:
Yeah I think we should avoid the HC middleware for now. It does make things easy but we had a lot of trouble with it in the intern house